src/test/java/org/takes/facets/auth/social/PsFacebookTest.java

```diff
@@ -52,15 +53,20 @@
     @Test
     public final void canLogin() throws Exception {
         final String identifier = RandomStringUtils.randomAlphanumeric(10);
+        final RandomStringGenerator generator =
```

Comment:
@krzyk I think that we could replace `RandomStringGenerator` and `RandomStringUtils` usage with `RandomText` from cactoos, WDYT?

Reply:
@paulodamaso But `RandomText` configuration is cumbersome, I would need to list all UTF characters there and do filtering, all this is done automatically byt RandomStringGenerator and RandomStringUtils.

Reply:
@krzyk `RandomText` comes with an pre-defined set of characters, so you'd just need to filter the unwanted ones, I think that it'll be enough for our test cases, WDYT?

Reply:
@paulodamaso OK, and are you OK with IntStream for generating the list of characters?

Reply:
@krzyk Yes, there is something that should I be worried about?

Reply:
@paulodamaso No, I was worried that I would need to embed zillions of classes from cactoos (because there is no direct way to generate a List or Iterator with provided seed and function) to get the same thing I can do in 4-5 lines with streams :)

Reply:
@paulodamaso So after reviewing it more, it is impossible to do. RandomText should accept a list of codepoints, but it accepts only Characters. One codepoint can result in mutliple chars (and the order is important here). And here we want to test unicode chars that can be outside of the simple 64k space that can be represented by a single char.

Reply:
@krzyk Okay, so that's nothing else to do here, thanks!